tools/clang/lib/CodeGen/CGHLSLMS.cpp

```diff
@@ -2435,6 +2435,9 @@ void CGMSHLSLRuntime::AddHLSLFunctionInfo(Function *F, const FunctionDecl *FD) {
             node.MaxRecords =
                 parmDecl->getAttr<HLSLMaxRecordsAttr>()->getMaxCount();
           }
+          if (parmDecl->hasAttr<HLSLGloballyCoherentAttr>()) {
+            node.Flags.SetGloballyCoherent();
+          }
```

Comment:
nit: LLVM style prefers omitting braces for single-line bodies (https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements).